Source/Public/ConvertTo-Bicep.ps1

```diff
@@ -21,22 +21,44 @@
     Go to module repository https://github.com/StefanIvemo/BicepPowerShell for detailed info, reporting issues and to submit contributions.
 #>
 function ConvertTo-Bicep {
+    [CmdletBinding()]
     param(
-        [string]$Path = $pwd.path
+        [string]$Path = $pwd.path,
+        
+        [string]$OutputDirectory = $pwd.path,
+
+        [switch]$AsString
     )
 
-    if (TestBicep) {
+    begin {
+        if (-not (Test-Path $OutputDirectory)) {
+            New-Item $OutputDirectory -Force -ItemType Directory
+        }
+        
+        $FileResolver = [Bicep.Core.FileSystem.FileResolver]::new()
+        $ResourceProvider = [Bicep.Core.TypeSystem.Az.AzResourceTypeProvider]::new()
+    }
+
+    process {
         $files = Get-Childitem -Path $Path *.json -File
         if ($files) {
             foreach ($file in $files) {
-                bicep decompile $file
-            }   
+                $BicepObject = [Bicep.Decompiler.TemplateDecompiler]::DecompileFileWithModules($ResourceProvider, $FileResolver, $file.FullName)
+                
```

Comment:
We should add the same warning message used by Bicep CLI when the command is excuted.
```
WARNING: Decompilation is a best-effort process, as there is no guaranteed mapping from ARM JSON to Bicep.
You may need to fix warnings and errors in the generated bicep file(s), or decompilation may fail entirely if an accurate conversion is not possible.
If you would like to report any issues or inaccurate conversions, please see https://github.com/Azure/bicep/issues.
```